### DISCUSSION TITLE
Horizon 7.4at issues

### DISCUSSION THREAD

**Discussion by molieriAC** [cagnulein/qdomyos-zwift#2916] [Q&A] [answered]
Hi, love the app. It's made my 2019 Horizon treadmill feel more fun by connecting to Zwift. The version I have is the model year before it got ftms (I think). I've got two issues that I've been unable to figure out a solution to.

1) If I need to pause during a workout the treadmill activity will no longer control Zwift once I restart. I've tried pausing in the app and with the treadmill. I've tried reconnecting before unpausing. Nothing works. My workaround is just to set the treadmill to the lowest speed possible then stepping away. Is there a setting that I can change or is that expected behavior? 

2) After 30 minutes the treadmill turns off. I believe this is tied to the default manual workout being 30 minutes. Is it possible to override this? 

Thanks for any help! Happy to try and get any logs or extra info if needed.

**Comment by cagnulein** [comment, marked as answer]
Hi @molieriAC try to enable this

![image](https://github.com/user-attachments/assets/ebb1b177-103a-46c2-bee8-f24ec9da1237)

And restart qz

**Reply by molieriAC** [reply]
That did it! It solved both problems. I never tried turning it on because I didn't have a 7.8.

Also, I apologize it looks like this was already discussed on the Facebook group. I didn't realize that there was one until after I'd asked the question here...

Thanks for all that you do!

**Reply by cagnulein** [reply]
please remember to rate the app on the store and if you can join patreon https://www.patreon.com/cagnulein
